elements/dat-import.js

```diff
@@ -64,19 +64,14 @@ function datImportElement (props) {
 
   assert.equal(typeof onsubmit, 'function', 'dat-import: onsubmit should be type function')
 
-  const linkIcon = icon({
-    id: 'link',
-    cls: 'absolute top-0 bottom-0 left-0'
-  })
-
   return html`
     <label for="dat-import" class="relative dib pa0 b--none ${prefix}">
       <input name="dat-import"
         type="text"
         placeholder="Import dat"
         onkeydown=${onKeyDown}
         class="input-reset">
-      ${linkIcon}
+      ${icon('link', { class: 'absolute top-0 bottom-0 left-0' })}
```

Comment:
Oh I kinda like that; very close to [hyperscript's class / id shorthand](https://github.com/hyperhype/hyperscript#classes--id) - I think having that at some point would definitely make sense. Nice :sparkles:

I'd probably make sure there'd be full consistency with hyperscript tho:
```js
icon('link.absolute.top-0.bottom-0.left-0#some-id')
```
but yeah that's nit picking. I like it!